lambdas/scripts/lambda_layer_update.py

```diff
@@ -0,0 +1,64 @@
+import re
+import sys
+
+import boto3
+
+
+class LambdaLayerUpdate:
+    def __init__(self):
+        self.client = boto3.client("lambda")
+
+        self.environment = sys.argv[1]
+        self.function_name_aws = f"{self.environment}_{sys.argv[2]}"
+        layers = sys.argv[3].strip().split(",")
+
+        self.updated_lambda_arns = {}
+        self.updated_layer_names = []
+
+        print(f"Function Name: {self.function_name_aws}")
+        print("Layers to add:")
+        for layer in layers:
+            updated_layer_name = f"{self.environment}_{layer}"
+            self.updated_layer_names.append(updated_layer_name)
+            print(f"- {updated_layer_name}")
+        print("")
+
+    def start(self):
+        self.extract_default_layer_arns()
+        self.extract_updated_layer_arns()
+        self.update_lambda()
+
+    def extract_default_layer_arns(self):
+        response = self.client.get_function(FunctionName=self.function_name_aws)
+        layer_arns_responses = response["Configuration"]["Layers"]
```

Comment:
```suggestion
        layer_responses = response["Configuration"]["Layers"]
```
As this is a list of layers not arns